fk/lisjp/src/main/java/se/inera/intyg/common/lisjp/v1/model/converter/InternalToCertificate.java

```diff
@@ -681,4 +772,866 @@ private static CertificateDataValueDiagnosis createDiagnosis(String id, Diagnos
+            )
+            .build();
+    }
+
+    private static CertificateDataElement createBehovAvSjukskrivningQuestion(LisjpUtlatandeV1 internalCertificate, int index) {
+        return CertificateDataElement.builder()
+            .id(BEHOV_AV_SJUKSKRIVNING_SVAR_ID_32)
+            .index(index)
+            .parent(BEDOMNING_CATEGORY_ID)
+            .config(
+                CertificateDataConfigSickLeavePeriod.builder()
+                    .text("Min bedömning av patientens nedsättning av arbetsförmågan")
+                    .description("Utgångspunkten är att patientens arbetsförmåga ska bedömas i förhållande till hens normala arbetstid.")
+                    .list(
+                        Arrays.asList(
+                            CheckboxDateRange.builder()
+                                .id(SjukskrivningsGrad.NEDSATT_1_4.getId())
+                                .label(SjukskrivningsGrad.NEDSATT_1_4.getLabel())
+                                .build(),
+                            CheckboxDateRange.builder()
+                                .id(SjukskrivningsGrad.NEDSATT_HALFTEN.getId())
+                                .label(SjukskrivningsGrad.NEDSATT_HALFTEN.getLabel())
+                                .build(),
+                            CheckboxDateRange.builder()
+                                .id(SjukskrivningsGrad.NEDSATT_3_4.getId())
+                                .label(SjukskrivningsGrad.NEDSATT_3_4.getLabel())
+                                .build(),
+                            CheckboxDateRange.builder()
+                                .id(SjukskrivningsGrad.HELT_NEDSATT.getId())
+                                .label(SjukskrivningsGrad.HELT_NEDSATT.getLabel())
+                                .build()
+                        )
+                    )
+                    .build()
+            )
+            .value(
+                CertificateDataValueDateRangeList.builder()
+                    .list(createSjukskrivningValue(internalCertificate))
+                    .build()
+            )
+            .validation(
+                new CertificateDataValidation[]{
+                    CertificateDataValidationMandatory.builder()
+                        .questionId(BEHOV_AV_SJUKSKRIVNING_SVAR_ID_32)
+                        .expression(multipleOrExpression(
+                            singleExpression(SjukskrivningsGrad.NEDSATT_1_4.getId()),
+                            singleExpression(SjukskrivningsGrad.NEDSATT_HALFTEN.getId()),
+                            singleExpression(SjukskrivningsGrad.NEDSATT_3_4.getId()),
+                            singleExpression(SjukskrivningsGrad.HELT_NEDSATT.getId())
+                        ))
+                        .build()
+                }
+            )
+            .build();
+    }
+
+    private static List<CertificateDataValueDateRange> createSjukskrivningValue(LisjpUtlatandeV1 internalCertificate) {
+        if (internalCertificate == null) {
```

Comment:
Maybe change the method argument to be a list of sjukskrivingar and make the null-check on it instead. We know for a fact that the internalCertificate isn't null.